virtcontainers/qemu.go

```diff
@@ -1152,8 +1153,7 @@ func (q *qemu) hotplugAddBlockDevice(drive *config.BlockDrive, op operation, dev
 			}
 		}()
 
-		// PCI address is in the format bridge-addr/device-addr eg. "03/02"
-		drive.PCIAddr = fmt.Sprintf("%02x", bridge.Addr) + "/" + addr
+		drive.PCIPath = vcTypes.PciPath{fmt.Sprintf("%02x", bridge.Addr) + "/" + addr}
```

Comment:
How about creating a `New()` function for `PciPath`? That would avoid the duplication here. Yes, it's trivial, but it's still duplication and thus could go stale if someone forgets to update one of the multiple calls at some point in the future.